hyppo/independence/hhg.py

```diff
@@ -18,6 +20,13 @@ class HHG(IndependenceTest):
     :footcite:p:`hellerConsistentMultivariateTest2013`. It can also operate on multiple
     dimensions :footcite:p:`hellerConsistentMultivariateTest2013`.
 
+    The fast version of this test can also be run, based on
```

Comment:
Add this to the notes section. Cite the paper by adding the bibtex to the refs.bib file

Reply:
Transferred to notes section.